src/cljs/main/broadfcui/page/method_repo/method_repo_table.cljs

```diff
@@ -59,32 +80,43 @@
                                                                    {:borderTop (when (pos? index) style/standard-line)
                                                                     :alignItems "center"})})
                              :columns
-                             [{:header "Method" :initial-width 300
-                               :column-data (juxt :namespace :name)
-                               :as-text (partial string/join "/")
-                               :sort-by (comp string/lower-case second)
+                             [{:id "certified" :initial-width 40
+                               :filterable? false :sortable? false :resizable? false
+                               :column-data
+                               (fn [{:keys [namespace method-id]}]
+                                 (or (contains? certified-namespaces namespace)
+                                     (contains? certified-methods method-id)))
+                               :as-text
+                               (fn [certified?]
+                                 (when certified? "Certified"))
+                               :render
+                               (fn [certified?]
+                                 (when certified? (icons/certified-icon {:style {:marginLeft -5}})))}
```

Comment:
I know you're just following the mockups, but after clicking around ... I can easily see the next request being "how can I get a list of just the certified methods?" A quick and dirty fix could be just setting `:sortable? false` (and giving the column a header)